.travis.yml

```diff
@@ -1,10 +1,26 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "4.4.2"
+  - '0.10'
+  - 4.4.2
 sudo: false
+services:
+  - docker
```

Comment:
Not sure why you are using a docker image here.

Travis has a sonarqube add on that seems to do it for you (https://docs.travis-ci.com/user/sonarqube/).

I might be missing something though.